models/appendix.py

```
@@ -5,6 +5,7 @@
 
 class Department(db.Model):
     __tablename__ = "setor"
+    __table_args__ = {"schema": "demo"}
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)